README.rst

```diff
@@ -27,14 +27,37 @@ Development environment
 
 From project root directory run::
 
-    docker-compose up
-    poetry run python manage.py migrate
+    docker-compose up -d
+
+Then we need to install pgloader (https://pgloader.readthedocs.io/en/latest/install.html#docker-images) and migrate MySQL database to PostgreSQL::
+
+    docker run -it --rm \
+    --network katalogas_default \
+    dimitri/pgloader:latest \
+    pgloader \
+        mysql://adp:secret@mysql/adp-dev \
+        postgresql://adp:secret@postgres/adp-dev
+
 
+By default pgloader creates a schema with the same name as in source database. So after this command we need to switch to public schema::
+
+    docker-compose run -T --rm -e PGPASSWORD=secret postgres psql -h postgres -U adp adp-dev <<EOF
+    BEGIN TRANSACTION;
+      ALTER SCHEMA "public" RENAME TO "public-orig";
+      ALTER SCHEMA "adp-dev" RENAME TO "public";
+      DROP SCHEMA "public-orig" CASCADE;
+    COMMIT;
+    EOF
+
+Then we can run migrations::
+
+    poetry run python manage.py migrate
```

Comment:
```suggestion
    poetry install
    poetry run python manage.py migrate
```

Before running this you need to run:

    poetry install

And this gives me:

    Installing psycopg2 (2.9.3): Failed

Because both `psycopg2` and `psycopg2-binary` are added into `pyproject.toml`.

    psycopg2 = "^2.9.3"
    psycopg2-binary = "^2.9.3"

You need to remove `psycopg2`.

And finally, `python manage.py migrate` also fails with the error:

    LINE 1: ALTER TABLE `adp_cms_attachment` MODIFY `deleted` TINYINT(1)...

Reply:
Removed failing migration, it was used to change the types of bit(1) fields to tinyint(1) in order to fix boolean fields in MySQL dababase. Now it's unnecessary, because after migrating MySQL database to PostgreSQL, types are fixed automatically - they are changed to boolean type.